go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/openshift/api
+
+go 1.13
+
+require (
+	github.com/emicklei/go-restful v2.11.1+incompatible // indirect
+	github.com/go-openapi/spec v0.19.4 // indirect
+	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
+	github.com/spf13/pflag v1.0.5
+	golang.org/x/net v0.0.0-20191207000613-e7e4b65ae663 // indirect
+	golang.org/x/tools v0.0.0-20191206204035-259af5ff87bd // indirect
+	gonum.org/v1/gonum v0.6.1 // indirect
+	gopkg.in/yaml.v2 v2.2.7 // indirect
+	k8s.io/api v0.0.0-20191204082340-384b28a90b2b
```

Comment:
@sttts @deads2k i think we want these pinned to kubernetes-1.16.2 (`replace ()`) right?

Reply:
In this case in hack/update-deps.sh should I have something like
```
go get -u=patch -d ./...
go mod tidy
go mod vendor
```
or should I remove this file completely?

Reply:
updated the PR to use `replace`: https://github.com/dmage/api/blob/gomod/go.mod